README.md

```diff
@@ -310,6 +310,7 @@ Questions can be asked by raising an `Issue`.
 - [Usman Tatla](https://github.com/iTatla1)
 - [Utkarsh Singh](https://github.com/Utkarsh299-tech/)
 - [Utkarsh Mishra](https://github.com/Utkarsh1504)
+- [Vicky Gupta](https://github.com/viki4545)
```

Comment:
```suggestion
```

Reply:
Instead move it under Vikas ganga's name

Reply:
done